scripts/import_historical_data.py

```
@@ -0,0 +1,266 @@
+            "websocket": 0,  # Can't determine from CSV
+            "email": 0,
+            "website": 0,
+        },
+        "by_language": dict(by_language),
+        "hourly_counts": dict(hourly_counts),
+        "daily_counts": dict(daily_counts),
+        "daily_earnings": {k: round(v, 2) for k, v in daily_earnings.items()},
+    }
+
+    # Print summary
+    print(f"\n--- Summary ---")
```

Ruff (0.15.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)